lib/AST/ASTImporter.cpp

```diff
@@ -4606,11 +4601,21 @@ Decl *ASTNodeImporter::VisitFunctionTemplateDecl(FunctionTemplateDecl *D) {
     }
   }
 
```

Comment:
The code from upper here was moved below this point. Is it because the previous code should not search in the TO context after putting something into it?